ui/text.go

```diff
@@ -62,6 +63,14 @@ func AlertInfo(text string) string {
 	return _aurora.Sprintf(GrayText(Bold("💁 %s\n").String()), text)
 }
 
+func AlertVerbose(isVerbose bool, text string) string {
```

Comment:
Nit: can we call this something like `InfoIfVerbose` or `VerboseInfo`?

The naming to me suggests that something will print, and you can add verbose to get WAY more. Maybe just me. Thoughts?

Reply:
That does make sense. I've renamed it to `VerboseInfo`.